pkg/cmd/checkendpoint/controller/status_updater.go

```diff
@@ -0,0 +1,75 @@
+package controller
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/cmd/checkendpoint/operatorcontrolplane/podnetworkconnectivitycheck/v1alpha1helpers"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/klog"
+)
+
+type StatusUpdater interface {
```

Comment:
what is a status updater? Missing high level doc.

Reply:
If I get it right, this queues up updateStatusFunc and calls them in batches every second?

Reply:
@sttts, correct. I have updated the godoc.